src/main/features/response/draft/responseDraft.ts

```diff
@@ -81,4 +81,18 @@ export class ResponseDraft extends DraftDocument implements Serializable<Respons
       && this.rejectPartOfClaim !== undefined
       && this.rejectPartOfClaim.option === RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED
   }
+
+  public requireMediation (): boolean {
+    if (!(this.response && this.response.type)) {
+      return false
+    }
+    if (this.response.type === ResponseType.OWE_NONE && this.rejectAllOfClaim &&
+      this.rejectAllOfClaim.option === RejectAllOfClaimOption.DISPUTE) {
+      return true
+    }
+    return this.response.type === ResponseType.OWE_SOME_PAID_NONE && this.rejectPartOfClaim &&
+      (this.rejectPartOfClaim.option === RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED ||
```

Comment:
`ResponseType.OWE_SOME_PAID_NONE` and `RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED` does not make much sense when you read it. `ResponseType` constants should be renamed to `ADMIT_ALL`, `REJECT_PART`, `REJEST_ALL` to improve this, but that would require updating Claim Store counterparts as well. Do you feel up for it @jacmurphy? 😉

Reply:
I completely agree with you. It would make it much clearer going forward. 
Absolutely! Tell me when you've changed it and ill update accordingly

Reply:
It's also a data migration too, I would say out of scope for this PR.
Either follow up PR or do when we save the partial admission work.

Reply:
@geek-soft I've got tests written but i cant tell if im getting false positives as the Free mediation is not appearing in the task list on correct input. I should have it sorted soon.